src/internal/m365/collection/drive/group_handler.go

```diff
@@ -27,16 +33,36 @@ func NewGroupBackupHandler(groupID string, ac api.Drives, scope selectors.Groups
 	}
 }
 
-func (h groupBackupHandler) CanonicalPath(
-	folders *path.Builder,
-	tenantID, resourceOwner string,
+func (h groupBackupHandler) PathPrefix(
+	tenantID, driveID string,
 ) (path.Path, error) {
-	// TODO(meain): path fixes
-	return folders.ToDataLayerPath(tenantID, h.groupID, h.service, path.LibrariesCategory, false)
+	// TODO: move tenantID to struct
+	return path.Build(
+		tenantID,
+		h.groupID,
+		h.service,
+		path.LibrariesCategory,
+		false,
+		odConsts.SitesPathDir,
+		h.siteID,
+		odConsts.DrivesPathDir,
+		driveID,
+		odConsts.RootPathDir)
 }
 
-func (h groupBackupHandler) ServiceCat() (path.ServiceType, path.CategoryType) {
-	return path.GroupsService, path.LibrariesCategory
+func (h groupBackupHandler) CanonicalPath(
+	folders *path.Builder,
+	tenantID string,
+) (path.Path, error) {
+	return folders.ToDataLayerPath(
```

Comment:
nit: I'd prefer manipulating parameters here instead of adding a vararg parameter to `ToDataLayerPath`. The surface area of the path package is already fairly large :/

Reply:
I'm not entirely sure I get the change you are suggesting. If you talking about just appending to the path here, the problem is that `folders.ToDataLayerPath` adds these as prefixes. Let me know if I am missing something obvious.